lib/index.js

```diff
@@ -987,29 +991,29 @@ module.exports = (function() {
         var self = this;
 
         var newCellsInserted = -1, // we technically delete one before we start adding back
-            currentCell = cell.attrib.r;
+        currentCell = cell.attrib.r;
 
             // add a cell for each element in the list
-        substitution.forEach(function(element) {
-            ++newCellsInserted;
+            substitution.forEach(function(element) {
+                ++newCellsInserted;
 
-            if(newCellsInserted > 0) {
-                currentCell = self.nextCol(currentCell);
-            }
+                if(newCellsInserted > 0) {
+                    currentCell = self.nextCol(currentCell);
+                }
 
-            var newCell = self.cloneElement(cell);
-            self.insertCellValue(newCell, element);
+                var newCell = self.cloneElement(cell);
+                self.insertCellValue(newCell, element);
 
-            newCell.attrib.r = currentCell;
-            cells.push(newCell);
-        });
+                newCell.attrib.r = currentCell;
+                cells.push(newCell);
+            });
 
-        return newCellsInserted;
-    };
+            return newCellsInserted;
+        };
 
     // Perform a table substitution. May update `newTableRows` and `cells` and change `cell`.
     // Returns total number of new cells inserted on the original row.
-    Workbook.prototype.substituteTable = function(row, newTableRows, cells, cell, namedTables, substitution, key, placeholder, drawing) {
+    Workbook.prototype.substituteTable = async function(row, newTableRows, cells, cell, namedTables, substitution, key, placeholder, drawing) {
```

Comment:
This is breaking change for consumers.

Reply:
I do agree it's a breaking change , so you will need to upgrade library version, instead of creating new functions behaves exactly the same